.github/workflows/publish.yml

```diff
@@ -104,7 +104,7 @@ jobs:
         with:
           context: ./src
           push: true
-          tags: ${{ secrets.ACR_PROD_SERVER }}/${{ steps.project-name.outputs.lowercase }}:${{ env.TAG }}
+          tags: ${{ secrets.ACR_PROD_SERVER }}/${{ steps.project-name.outputs.lowercase }}:${{ env.TAG }},latest
```

Comment:
We should also prefix latest like this
`${{ secrets.ACR_PROD_SERVER }}/${{ steps.project-name.outputs.lowercase }}:latest`